### DISCUSSION TITLE
Zmiana domeny na domenę federacyjną

### DISCUSSION THREAD

**Discussion by jaadmin** [AzureCommunityPL/group#361] [Pomoc] [answered]
Cześć, 
Próbuję zrobić jedną domenę na office365 na sfederowaną. 
po wydaniu polecenia Set-MsolDomainAuthentication -DomainName: *.pl -Authentication: Federated
dostaję Set-MsolDomainAuthentication : Unable to complete this action. Try again later.
Czy ktoś może spotkał się z takim zagadnieniem?

**Comment by eklime** [comment, marked as answer]
8 dni później - spróbowałeś kolejny raz tak jak sugerował komunikat?

**Reply by jaadmin** [reply]
Tak, udało się, składnię uzupełniłem dokładnie tak jak jest na stronie ms. Możliwe że gdzieś biały znak mi się pojawił.
